payloads/v10/channel.ts

```diff
@@ -688,6 +689,14 @@ export enum MessageType {
 	GuildInviteReminder,
 	ContextMenuCommand,
 	AutoModerationAction,
+
+	InteractionPremiumUpsell = 26,
+	StageStart,
+	StageEnd,
+	StageSpeaker,
+	StageRaiseHand,
+	StageTopic,
```

Comment:
Those need to be removed, ref: https://github.com/discord/discord-api-docs/pull/5826